### ISSUE TITLE
pacaur -k not limited to the AUR

### ISSUE THREAD

**Issue by wedeluxe** [rmarquis/pacaur#716]
###### Version

A couple of days ago I upgraded from version 4.6.10 to version 4.7.9.

###### Description

In version 4.6.10 `pacaur -k` listed all updates available in the AUR. This is expected behavior.

In version 4.7.9 `pacaur -k` lists all updates available in the AUR prefixed by available updates in the binary repos.

I read through all commit messages in between the two pacaur versions mentioned above, but couldn't find any hint towards an intended change of behavior.

###### Output

`pacaur -k`

This is only an excerpt form the output:
```
[...]
:: community  virtualbox                     5.1.18-1                    ->  5.1.22-2                                                    
:: community  virtualbox-guest-iso           5.1.18-1                    ->  5.1.22-1                                                    
:: community  virtualbox-host-modules-arch   5.1.18-3                    ->  5.1.22-2                                                    
:: extra      vlc                            2.2.4-8                     ->  2.2.4-9                                                     
:: extra      vte-common                     0.46.1-1                    ->  0.48.3-1                                                    
:: extra      vte3                           0.46.1-1                    ->  0.48.3-1                                                    
:: extra      webkit2gtk                     2.14.5-1                    ->  2.16.2-1                                                    
:: extra      webkitgtk                      2.4.11-4                    ->  2.4.11-6                                                    
:: extra      webkitgtk2                     2.4.11-4                    ->  2.4.11-6                                                    
:: extra      wildmidi                       0.4.0-1                     ->  0.4.1-1                                                     
:: multilib   wine-staging                   2.5-1                       ->  2.7-1                                                       
:: community  winetricks                     20170327-1                  ->  20170517-1                                                  
:: core       wpa_supplicant                 1:2.6-4                     ->  1:2.6-5                                                     
:: extra      wxgtk                          3.0.2-6                     ->  3.0.3.1-1                                                   
:: extra      x265                           2.3-1                       ->  2.4-1                                                       
:: extra      xdg-utils                      1.1.1-5                     ->  1.1.2-1                                                     
:: extra      xf86-input-libinput            0.25.0-1                    ->  0.25.1-1                    (xorg-drivers)                  
:: extra      xf86-video-intel               1:2.99.917+770+gcb6ba2da-1  ->  1:2.99.917+772+gc72bb27a-1  (xorg-drivers)                  
:: extra      xorg-xkbcomp                   1.3.1-1                     ->  1.4.0-1                     (xorg-apps xorg)                
:: community  zathura-pdf-mupdf              0.3.1-1                     ->  0.3.1-3                                                     
:: extra      zenity                         3.22.0+1+gcd1647c-1         ->  3.24.0-1                                                    
:: aur        copyq                          2.7.1-1                     ->  3.0.2-1                                                     
:: aur        cower                          16-1                        ->  17-2                                                        
:: aur        geoclue                        0.12.99-2                   ->  0.12.99-3                                                   
:: aur        gnome-perl                     1.045-4                     ->  1.045-8                                                     
:: aur        gnome-vfs-perl                 1.082-4                     ->  1.082-7                                                     
:: aur        gnomecanvas-perl               1.002-14                    ->  1.002-18                                                    
:: aur        goocanvas1                     1.0.0-3                     ->  1.0.0-4                                                     
:: aur        google-chrome                  57.0.2987.98-1              ->  59.0.3071.115-1                                             
:: aur        grantlee-qt4                   0.5.1-3                     ->  0.5.1-4                                                     
:: aur        gstreamer0.10-bad              0.10.23-20                  ->  0.10.23-36                                                  
:: aur        gstreamer0.10-bad-plugins      0.10.23-20                  ->  0.10.23-36                                                  
:: aur        gstreamer0.10-base             0.10.36-3                   ->  0.10.36-9                                                   
:: aur        gstreamer0.10-base-plugins     0.10.36-3                   ->  0.10.36-9                                                   
:: aur        gstreamer0.10-good             0.10.31-11                  ->  0.10.31-27                                                  
:: aur        gstreamer0.10-good-plugins     0.10.31-11                  ->  0.10.31-27                                                  
:: aur        gstreamer0.10-ugly             0.10.19-16                  ->  0.10.19-27                                                  
:: aur        gstreamer0.10-ugly-plugins     0.10.19-16                  ->  0.10.19-27                                                  
:: aur        gtkhtml4                       4.8.5-1                     ->  4.10.0-1                                                    
:: aur        jdk                            8u121-1                     ->  8u131-1                                                     
:: aur        kodi-platform-git              r38.15edaf7-1               ->  latest                                                      
:: aur        komodo-edit                    10.1.4_17456-1              ->  10.2.2_17703-2                                              
:: aur        makemkv                        1.10.4-1                    ->  1.10.6-1                                                    
:: aur        menulibre                      2.1.0-1                     ->  2.1.3-2                                                     
:: aur        multibootusb                   7.5.0-1                     ->  8.8.0-3                                                     
:: aur        musicbrainz                    2.1.5-7                     ->  2.1.5-8                                                     
:: aur        perl-gnome2-wnck               0.16-11                     ->  0.16-15                                                     
:: aur        perl-goo-canvas                0.06-7                      ->  0.06-10                                                     
:: aur        perl-gtk2-imageview            0.05-7                      ->  0.05-10                                                     
:: aur        perl-gtk2-unique               0.05-13                     ->  0.05-18                                                     
:: aur        pipelight                      1:0.2.8-3                   ->  1:0.2.8.2-2                                                 
:: aur        prelink                        20130503-5                  ->  20130503-7                                                  
:: aur        python-path                    8.2.1-1                     ->  10.1-1                                                      
:: aur        qtwebkit                       2.3.4-4                     ->  2.3.4-5                                                     
:: aur        radeon-profile-git             20161221.r13.g4385106-1     ->  latest                                                      
:: aur        radeontop                      1.0-1                       ->  1.0-5                                                       
:: aur        roger-router                   1.8.12-1                    ->  1.9.3-2                                                     
:: aur        shutter                        0.93.1-3                    ->  0.93.1-11                                                   
:: aur        virtualbox-ext-oracle          5.1.18-1                    ->  5.1.22-1                                                    
:: aur        wxglade                        0.7.0-3                     ->  0.7.2-1      
```

**Comment by rmarquis** [comment]
Indeed unexpected. Thanks for the report, I'll have a look asap.

**Comment by rmarquis** [comment]
This regression has been introduced in 7b1eb8b92fd7035028dcd596c7acca81dd467e8e. Fixed in 71294f470519b40023bcab84c58aa109e43b4ca3.
